task/02-numbers-tasks.js

```diff
@@ -54,7 +54,8 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+    var res = ((BigInt(value1)+BigInt(value2))/BigInt(2));
+   return res;
```

Comment:
Cool! Unexpected solution =) nodejs  supports BigInt from 10.4 version. And the solution is really dangerous for browsers.
Please try to solve the issue without BigInt also

Reply:
fixed